### PR TITLE
chore: student attendance sync rule and seeder

### PR DESCRIPTION
**Pull request by Kelv1nG**
🎫 Ticket

[chore: student attendance seeder](https://trello.com/c/KBZLj1HS/96-student-attendance-seeder)

📦 Packages

none

📝 Updates

add student attendance seeder
- attendance for every student of every school only for current academic year ( weekends are excluded)
- absents / lates are randomized

update attendance table
- changed checked_by field in attendances to be nullable
- changed academic_year_id to section_id for syncing reasons

🧪 Testing Steps

N/A

### CODE REVIEW

File: config/sync_rules.yaml [hunk 1]
```diff
@@ -161,6 +181,7 @@ bucket_definitions:
 
       - SELECT * 
         FROM assessment_questions
+
```
Comment (ejempic) [inline config/sync_rules.yaml:184]: remove

File: src/app/Enums/AttendanceStatus.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+namespace App\Enums;
+
+//TODO:: this needs to be separated for a student record, or the teacher assessment record
+enum AttendanceStatus: string
+{
+    case PRESENT = 'present';
+    case LATE = 'late';
+    case ABSENT = 'absent';
+    case AUTHORIZED_ABSENT = 'to_finish_evaluation';
```
Comment (ejempic) [inline src/app/Enums/AttendanceStatus.php:11]: used for?

Comment (Kelv1nG) [reply src/app/Enums/AttendanceStatus.php:11]: this is used in the 'attendances' table/migration in 'status' field for explicitness

File: config/sync_rules.yaml (hunk 1 above)
Comment (Kelv1nG) [reply config/sync_rules.yaml]: removed

**Pull request merged**